app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/DatasetOverViewFragment.kt

```diff
@@ -0,0 +1,66 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+
+/**
+ * A fragment representing a list of Items.
+ */
+class DatasetOverViewFragment : Fragment(), DatasetOverviewRecyclerViewAdapter.OnItemClickListener {
+
+    private var columnCount = 1
```

Comment:
Consider making this a lateinit var instead, unless you really need this default value?
```suggestion
    private lateinit var columnCount : Int
```